Makefile

```diff
@@ -82,6 +82,10 @@ cmake:					#: Use CMake to create a Makefile build system
 build:					#: Build the software based in BUILD_DIR and BUILD_TYPE variables
 	cmake --build $(BUILD_BASE_DIR)/$(BUILD_DIR) -j $(NUM_THREADS)
 
+substrait:          	#: Build with debugging symbols
+	$(MAKE) cmake BUILD_DIR=debug BUILD_TYPE=Debug EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_SUBSTRAIT=ON"
+	$(MAKE) build BUILD_DIR=debug -j ${NUM_THREADS}
+
```

Comment:
I hope this is okay to add?

Reply:
@vibhatha This change should go into a separate PR. BTW, what's the motivation for this change?

Reply:
Basically, just to build the Substrait things this and kind of easier to deal with. No special motivation here. 🙂